omero/users/whatsnew.txt

```diff
@@ -1,16 +1,18 @@
-What's new for OMERO 5.2 for users
+What's new for OMERO 5.3 for users
 ==================================
 
-Updates and new features for OMERO 5.2 include:
+Updates and new features for OMERO 5.3 include:
 
--  improved support for screen data formats (amongst others) and import times
-   for images with many ROIs
--  reworked display of metadata and annotations in the right hand pane of both
-   clients
--  improved support for attaching analytical results
--  significant improvement in client performance including for loading data
-   and drag and drop actions
--  the size limit for exporting images is now 12k x 12k pixels
+- ROI Folders which allow you to organize your ROIs into a hierarchical tree
+  structure of folders so they can be sorted e.g. by phenotypes or ontologies
+- support for lookup tables (LUT) and reverse intensity rendering
+- annotations can now be added to Wells and the UI for viewing Screen Plate
+  Well data has been redesigned
+- rendering engine improvements mean that full projection thumbnails are now
+  available
+- the new 'Open With...' functionality allows OMERO.figure or other plugin
+  custom viewers to be launched directly from OMERO.web (coming soon is a new
```

Comment:
the term "viewers" might be misleading since I can see for example ``mapr`` being a plugin to use in an "open with" way